src/Logic/Utilities.cs

```diff
@@ -792,14 +792,25 @@ private static string ReInsertHtmlTags(string s, Dictionary<int, string> htmlTag
 
         public static string UnbreakLine(string text)
         {
-            if (text.Contains(Environment.NewLine))
+            if (!text.Contains(Environment.NewLine))
+                return text;
+
+            var singleLine = text.Replace(Environment.NewLine, " ");
+            while (singleLine.Contains("  "))
+                singleLine = singleLine.Replace("  ", " ");
+
+            if (singleLine.Contains("</")) // Fix tag
             {
-                string newText = text.Replace(Environment.NewLine, " ");
-                while (newText.Contains("  "))
-                    newText = newText.Replace("  ", " ");
-                return newText;
+                singleLine = singleLine.Replace("</i> <i>", " ");
+                singleLine = singleLine.Replace("</i><i>", " ");
+
+                singleLine = singleLine.Replace("</b> <b>", " ");
+                singleLine = singleLine.Replace("</b><b>", " ");
+
+                singleLine = singleLine.Replace("</u> <u>", " ");
+                singleLine = singleLine.Replace("</u><u>", " ");
             }

```

Comment:
Useful when *user* just want to *unbreak* line without running the **Fix Common Errors**